src/App.css

```diff
@@ -36,3 +30,16 @@
     transform: rotate(360deg);
   }
 }
+
+.home-page {
+  display: grid;
+  grid-template-columns: 5% 1fr 5%;
+  background-color: darkred;
+}
+
+.card-section {
```

Comment:
Awesome, getting that styling in. I was thinking last night we may want to add a container component for the posters. We can talk about everything in our stand up. Great work as always. Talk to you soon.